pkg/deploy/identity-provider/keycloak_checks.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (c) 2012-2019 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package identity_provider
+
+import (
+	"fmt"
+
+	"github.com/eclipse-che/che-operator/pkg/deploy"
+	"github.com/eclipse-che/che-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func GetWaitForKeycloakInitContainer(deployContext *deploy.DeployContext) (*corev1.Container, error) {
+	keycloakEndpointCheckerImage := deploy.DefaultKeycloakImage(deployContext.CheCluster)
```

Comment:
To be consistent let's take an image from keycloak deployment like in postgres case.

Reply:
I see no need in it. In case of Postgres we need to be consistent with its versions, but here there is no such a requirement.